CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 * Fix Rhino7 Mac installation path
+* Separate `compas.robots.Joint.origin` into the parent-relative origin `origin` and the world-relative origin `current_origin`.
```

Comment:
I like the `parent-relative vs world-relative` wording, maybe add as inlined comment at its first occurrence.